draft-ietf-quic-transport.md

```diff
@@ -795,13 +794,21 @@ COPT (0x0005):
   cases include changing congestion control algorithms and parameters such as
   initial window.  (TODO: List connection options.)
 
+
 ### Values of Transport Parameters for 0-RTT
 
-Transport parameters from the server are remembered by the client for 0-RTT
-connections.  If values change as a result of completing the handshake, the
-client is expected to respect the new values.  This introduces some potential
-problems, particularly with respect to transport parameters that establish
-limits:
+Optional transport parameters from the server MUST be reset to default values by
+the client for 0-RTT connections.
+
+The mandatory transport parameters for stream and connection flow control and
+concurrent stream limits MUST be remembered by the client.  The idle timeout value
```

Comment:
The problem with "MAY" is that we don't establish defaults.  If the client decides not to remember it has no idea how much flow control window it can use or how many streams it can open.  So we either need to define defaults (maybe 0-RTT defaults) or we do this as a MUST.